File: script/python_pro/ecart_type.py
from fonction import *
import math


tab = []

print("Ce programme calcule l'écart-type expérimental.\n")
print("Le calcul de l'écart-type ne concerne que les tableaux contenant au moins deux valeurs.\n")

e = int(input("Combien de nombres voulez-vous calculer leur écart-type ? "))
	
	
if e >= 2:
    print("Entrez à tour de rôle les nombres de votre tableau.\n")
    print("NB : pour les nombres à virgule, utilisez le point (ex : 12.5).")
    for i in range(e):
        a = float(input(f"N°{i+1} = "))
        tab.append(a)

    ecart_type(e, tab)

else:
    print ("Il faut au minimum deux valeurs pour executer ce programme")

File: script/c_pro/ecart_type.c
#include <stdio.h>
#include "fonction.h"
int main ()
{

        int e;
        printf("Ce programme est utilisé pour calculer l'écart-type selon la physique.");
        printf("Le calcul de l'écart-type ne concerne que les tableaux contenant au moins deux valeurs.");

        puts ("Combien de nombres voulez-vous calculer leur écart-type ?");
        scanf ("%d",&e);
        if (e >= 2)
        {

                printf("Entrez à tour de rôle les nombres de votre tableau :");
                printf("NB : pour les nombres à virgule, utilisez le point (.) comme séparateur.");
                double tab[e];
                for (int i = 0; i < e; i++)
                {
                        scanf("%lf", &tab[i]);
                }
                ecart_type(e, tab);
                return 0;
        }
        else
        {
                printf("Il faut au minimum deux valeurs pour executer ce programme");
                return 1;
        }
}
